collectors/apps.plugin/apps_groups.conf

```
@@ -314,3 +314,15 @@ p4: p4*
 git-services: gitea gitlab-runner
 
 freeswitch: freeswitch*
+
+# -------- web3 / blockchains ----------
+
+ethereum node: *geth* *openethereum*
+urbit: urbit*
+bitcoin node: *bitcoind* lnd*
```

```suggestion
bitcoin-node: *bitcoind* lnd*
```

I did a quick test and having space in a group name not really _allowed_. The group name would be `bitcoin`, not `bitcoin node`. 

I see [it is possible to have spaces in the processes names](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#configuration) (the part after `:`):

I tried using `"` and `'` in the group name with space - no luck.

> To add process names with spaces, enclose them in quotes (single or double) example: 'Plex Media Serv' or "my other process"

---

Then we should remove that quote from the config and have it say about using `-` instead of a space